sttp/Subscriber.go

```diff
@@ -676,6 +687,7 @@ func (sb *Subscriber) handleProcessingComplete(message string) {
 	sb.beginCallbackSync()
 
 	if sb.historicalReadCompleteReceiver != nil {
+		sb.traceCallback(sb.historicalReadCompleteReceiver, "calling historicalReadCompleteReceiver")
 		sb.historicalReadCompleteReceiver()
```

Comment:
Is this going to call this callback twice?

Reply:
Good catch.